modules/data.land/inst/IC_prep_anchorSites.Rmd

```diff
@@ -0,0 +1,158 @@
+---
+title: "Initial condition prep script for NA anchor sites"
```

Comment:
feel like this script should probably be in the inst/anchor folder.

Also, as a general comment (not just this script), if you're going to do prep in Rmd instead of R, I'd recommend embedding more text-based descriptions and explanations. Think about the next cohort of folks to use these tools and what they need to know that's still in your head instead of written out explicitly. 

On this point, also a reminder to think about what in this PR also needs to be added to the general pecan documentation and change logs. For example, there's whole new sources of input data that need to be included. Any new pecan.xml tags also need to be documented

Reply:
Fixed.

Reply:
I see this script was moved.

Still need to address the need for updates to /book_source